kuma/users/views.py

```diff
@@ -644,3 +644,11 @@ def recover(request, uidb64=None, token=None):
 
 recover_done = login_required(never_cache(ConnectionsView.as_view(
     template_name='users/recover_done.html')))
+
+
+def logout(request):
```

Comment:
I don't see why allauth's [logout view](https://github.com/pennersr/django-allauth/blob/0.24.1/allauth/account/views.py#L634) should be reimplemented.